integration/apps/rack/Dockerfile-ci

```diff
@@ -8,6 +8,11 @@ COPY . /vendor/dd-trace-rb
 # Install dependencies
 ENV DD_DEMO_ENV_GEM_LOCAL_DDTRACE /vendor/dd-trace-rb
 
+# Override number of concurrent compiles in grpc gem, see https://github.com/grpc/grpc/pull/28250 and https://github.com/DataDog/dd-trace-rb/issues/1791
+# If you see gem installation failing with "Killed" on CircleCI and `gem install --platform ruby grpc` reproduces the
+# issue when you connect to the testing container via ssh, then try lowering this file a notch.
+ENV GRPC_RUBY_BUILD_PROCS: 6
```

Comment:
(We talked about this privately, but just for visibility I'm adding it here as well)

I believe this (and all similar changes) are no longer needed after we removed all references to grpc in the integration apps :)

Reply:
Reverted